content/design/vendoring.md

```diff
@@ -53,3 +53,224 @@ already written):
 You now have a `vendor` directory that contains the sources for `goagen`, compile them and use the
 tool as described above. Don't forget to recompile the tool after each `glide update` that updates
 `goa`.
+
+# Code-based generation and vendoring
+
+> For the impatients, here is a concise version of what we'll see here:
+>
+> ```
+> $ mkdir -p $GOPATH/src/github.com/your/pkg/design
+> $ cd !$
+> $ # Create your design in the `design` package
+> $ # Paste de content of `gen/main.go` below to `...github.com/your/pkg/gen/main.go`
+> $ go run gen/main.go      # generates all the things
+> $ govendor init           # creates the vendor/ dir
+> $ govendor add +e         # pull needed packages in
+> ```
+>
+> And you're set. Everyone in your team will be able to deterministically
+> regenerate your code if they tweak the design, without version conflicts.
+
```

Comment:
Maybe something like:

We'll create a `gen` tool to that invokes the `goa` code generation function: